build/webpack.client.dev.js

```diff
@@ -27,26 +26,18 @@ const {
 } = plugins;
 const {
   withStylingModuleLoader,
-  withTSModuleLoader,
-  withJSModuleLoader,
   withFontModuleLoader,
   withImageModuleLoader,
+  withTypeScriptModuleLoader,
```

Comment:
Both modules were duplicate code but one set to test for `.js` and other `.ts` - merged the two as the tsconfig is set to compile js too.